LibreHardwareMonitorLib/Hardware/Motherboard/Identification.cs

```diff
@@ -273,6 +273,15 @@ public static Model GetModel(string name)
             case var _ when name.Equals("Z270 PC MATE", StringComparison.OrdinalIgnoreCase):
             case var _ when name.Equals("Z270 PC MATE (MS-7A72)", StringComparison.OrdinalIgnoreCase):
                 return Model.Z270_PC_MATE;
+            case var _ when name.Equals("Z77 MPower", StringComparison.OrdinalIgnoreCase):       // MS-7751 Rev 4.x
+            case var _ when name.Equals("Z77 MPower (MS-7751)", StringComparison.OrdinalIgnoreCase):
+            case var _ when name.Equals("Z77A-GD65", StringComparison.OrdinalIgnoreCase):        // MS-7751 Rev >1.x
+            case var _ when name.Equals("Z77A-GD65 (MS-7751)", StringComparison.OrdinalIgnoreCase):
+            case var _ when name.Equals("Z77A-GD65 GAMING", StringComparison.OrdinalIgnoreCase): // MS-7751 Rev 2.x
+            case var _ when name.Equals("Z77A-GD65 GAMING (MS-7751)", StringComparison.OrdinalIgnoreCase):
+            case var _ when name.Equals("Z77A-GD55", StringComparison.OrdinalIgnoreCase):        // MS-7751 Rev 1.x
+            case var _ when name.Equals("Z77A-GD55 (MS-7751)", StringComparison.OrdinalIgnoreCase):
+                return Model.Z77_MPower;
```

Comment:
If I may, why not use something more generic like `Model.Z77_MS7751` instead?

Reply:
Its "legacy". Platform is the same, some Z68 boards, that are not an MS-7751 also have the same logic. I think that variable name is not so important.

Reply:
Also the same code works for the "Z68A-GD80 (MS-7757)" and similiar MSI motherboards with non Intel XE chipset and with F71889AD. This code I will PR later, and it is available in my fork (master).